src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,101 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final String ROW_SUPPLY = "supply";
+    static final String ROW_BUY = "buy";
+    static final String ROW_RESULT = "result";
+    static final String SEPARATOR = ",";
+    static final int RESET_DATA = 0;
+    static final int INDEX_FIRST = 0;
+    static final int INDEX_SECOND = 1;
+    private int supply = 0;
+    private int buy = 0;
+
+    public int getSupply() {
+        return supply;
+    }
+
+    public void setSupply(int supply) {
+        this.supply = supply;
+    }
+
+    public int getBuy() {
+        return buy;
+    }
+
+    public void setBuy(int buy) {
+        this.buy = buy;
+    }
```

Comment:
this isn't a model, it's a service, we don't need getters setters for fields, moreover, they shouldn't be fields in the first place :)